prelexer.cpp

```diff
@@ -262,10 +262,9 @@ namespace Sass {
       return word<else_kwd>(src);
     }
     const char* elseif_directive(const char* src) {
-      return alternatives< word<else_kwd>,
-                           sequence< exactly< else_kwd >,
-                                     optional_css_whitespace,
-                                     word< if_after_else_kwd > > >(src);
+      return sequence< exactly< else_kwd >,
+                                optional_css_whitespace,
+                                word< if_after_else_kwd > >(src);
```

Comment:
Shouldn't it be (untested):
```
return word <
         sequence<
           exactly< else_kwd >,
           optional_css_comments,
           exactly< if_after_else_kwd >
         >
       >(src);
```

The `word` matcher only checks if it is not followed by a `alnum` character. So it doesn't care if the thing inside has any spaces (I guess this might be confusing). IMO your solution does exactly the same, but IMO indentation is off. And btw. I don't know, but I actually prefer to write each lexer on its own line, it's much better readable to me, specially with the correct indentation. Just wanted to ask as I tend to re-write them and wanted to get consens!?